### DISCUSSION TITLE
Publicly share ebs snapshots of the postgres and flatnode folders

### DISCUSSION THREAD

**Discussion by meyerovb** [mediagis/nominatim-docker#434] [Ideas]
Is it allowed for me to publicly share aws ebs snapshots of the two directories so anyone else can just restore them, spin up a docker and point it at them, and be off and running without needing to import data?

**Comment by leonardehrenfried** [comment]
Yes, that's fine. The data is all public so I can't see a reason why you shouldn't do that.

**Reply by meyerovb** [reply]
Where can I ask for someone to do that for me, for a fee?

**Reply by leonardehrenfried** [reply]
Freelancer websites?
